src/Raven.Server/Documents/TcpHandlers/SubscriptionConnection.cs

```diff
@@ -150,12 +152,16 @@ public static void SendSubscriptionDocuments(DocumentDatabase database, JsonOper
                     {
                         if (tcpClient == null || tcpClient.Connected == false)
                             return;
+
+                        if (connection._options.ConnectionException != null)
+                            return;
                         using (var writer = new BlittableJsonTextWriter(context, stream))
                         {
                             context.Write(writer, new DynamicJsonValue
                             {
-                                ["Type"] = "Error",
```

Comment:
We want to have the same types of messages everywhere, revert this to use the same Type/Exception as before. If you need more info, add another field.